lucet-runtime/lucet-runtime-tests/src/timeout.rs

```diff
@@ -100,10 +100,13 @@ macro_rules! timeout_tests {
 
             let kill_switch = inst.kill_switch();
 
-            thread::spawn(move || {
-                thread::sleep(Duration::from_millis(100));
-                assert_eq!(kill_switch.terminate(), Ok(KillSuccess::Signalled));
-            });
+            thread::Builder::new()
+                .name("helper".to_owned())
```

Comment:
Updated these tests to use [thread::Builder](https://doc.rust-lang.org/std/thread/struct.Builder.html) rather than the free `thread::spawn` function so that we can name the threads.

This makes for large quality of life improvements when debugging problems with these tests.